target/linux/qualcommax/ipq807x/base-files/etc/hotplug.d/firmware/11-ath11k-caldata

```diff
@@ -13,6 +13,8 @@ case "$FIRMWARE" in
 	compex,wpq873|\
 	edgecore,eap102|\
 	edimax,cax1800|\
+	linksys,mx4200v1|\
+	linksys,mx4200v2|\
```

Comment:
You should keep it in alphabetical order.

Reply:
The resolution causes a merge conflict